parsl/providers/provider_base.py

```diff
@@ -26,9 +26,13 @@ def __new__(cls, value, terminal):
 class JobStatus(object):
     """Encapsulates a job state together with other details, presently a (error) message"""
 
-    def __init__(self, state: JobState, message: str = None):
+    def __init__(self, state: JobState, message: str = None, exit_code: int = -1,
```

Comment:
use None for exit_code. don't invent one when one doesn't exist.

Reply:
Done